src/viur/core/bones/boolean.py

```diff
@@ -18,13 +18,14 @@ class BooleanBone(BaseBone):
     def __init__(
         self,
         *,
-        defaultValue: bool | list[bool] | dict[str, list[bool] | bool] = None,
+        defaultValue: bool | list[bool] | dict[str, list[bool] | bool] | t.Callable = None,
         **kwargs
     ):
         if defaultValue is not None:
             # We have given an explicit defaultValue and maybe a complex structure
-            if not (kwargs.get("multiple") or kwargs.get("languages")) and not (isinstance(defaultValue, bool)):
-                raise TypeError("Only 'True', 'False' or 'None' can be provided as BooleanBone defaultValue")
+            if not kwargs.get("languages") and not (isinstance(defaultValue, bool) and not callable(defaultValue)):
+                raise TypeError("Only 'True', 'False', 'None' or 'callable' "
+                                "can be provided as BooleanBone defaultValue")
```

Comment:
This does not work for me:
```py
    test_boolean_callable_single_lang1 = BooleanBone(
        defaultValue=lambda skel, bone: print(skel["key"], bone.descr) or bool(random.randint(0,1)),
    )
    test_boolean_callable_multi_lang1 = BooleanBone(
        languages=["de", "en", "fr"],
        defaultValue=lambda skel, bone: print(skel["key"], bone.descr) or bool(random.randint(0,1)),
    )
    test_boolean_callable_multi_lang2 = BooleanBone(
        languages=["de", "en", "fr", "nl"],
        defaultValue=lambda skel, bone: {"de": True, "en": False, "fr": None},
    )
```



```
    test_boolean_callable_single_lang1 = BooleanBone(
                                         ^^^^^^^^^^^^
  File "/home/.../deploy/viur/core/bones/boolean.py", line 29, in __init__
    raise TypeError("Only 'True', 'False', 'None' or 'callable' "
TypeError: Only 'True', 'False', 'None' or 'callable' can be provided as BooleanBone defaultValue
```


---
Shouldn't this be

```suggestion
            if not kwargs.get("languages") and not (isinstance(defaultValue, bool) or callable(defaultValue)):

```